src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -15,20 +15,33 @@
 // </copyright>
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using OpenTelemetry.Metrics;
-using OpenTelemetry.Resources;
 
 namespace OpenTelemetry.AutoInstrumentation.Configuration;
 
 internal static class EnvironmentConfigurationMetricHelper
 {
+    private static readonly Dictionary<MeterInstrumentation, Action<MeterProviderBuilder>> AddMeters = new()
+    {
+        [MeterInstrumentation.AspNet] = builder => builder.AddRuntimeMetrics(),
```

Comment:
1. This should be  `[MeterInstrumentation.NetRuntime] = builder => builder.AddRuntimeMetrics(),`. These are .NET Runtime metrics - not ASP.NET metrics.
2. The ASP.NET metrics could be added similarly to ASP.NET tracing as both [`OpenTelemetry.Instrumentation.AspNetCore`](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore) and [`OpenTelemetry.Instrumentation.AspNet`](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNet) support metrics

Reply:
Good catch!
Fixed 6fe678e61fb86c6c166cd5b392b6485558046e34.